app/models/user.js

```diff
@@ -40,5 +43,9 @@ export default Addressable.extend({
 
   fullName: function(){
     return (this.get('firstName') + " " + this.get('lastName'));
-  }.property('firstName', 'lastName')
+  }.property('firstName', 'lastName'),
+
+  lName: function(){
```

Comment:
Let's alter the code so it uses '' instead of null for empty last names. Also, with the changes I've made to the API, the system user now has last name "Team"